src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/Microsoft.DotNet.UpgradeAssistant.Steps.Solution.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\common\Microsoft.DotNet.UpgradeAssistant.Abstractions.Internal\Microsoft.DotNet.UpgradeAssistant.Abstractions.Internal.csproj" />
```

Comment:
What was this needed for? Feature flags or WorkspaceOptions, I expect, but I didn't see where the solution step used those.

Reply:
I moved the graph traversal stuff there